src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java

```diff
@@ -252,14 +257,30 @@ private ReplayComparisonConfig getCompareConfig(ReplayActionItem actionItem) {
         return compareConfigService.loadConfig(actionItem);
     }
 
-    private CompareOptions buildCompareRequest(ReplayComparisonConfig compareConfig) {
+    private CompareOptions buildCompareRequest(String category, ReplayComparisonConfig compareConfig) {
         CompareOptions options = new CompareOptions();
+        options.putCategoryType(convertCategoryType(category));
+        // todo: the switch of "sqlBodyParse" and "onlyCompareCoincidentColumn"
+        //  need get from ReplayComparisonConfig
+        options.putSqlBodyParse(true);
+        options.putOnlyCompareCoincidentColumn(true);
         options.putExclusions(compareConfig.getExclusionList());
         options.putInclusions(compareConfig.getInclusionList());
         options.putListSortConfig(compareConfig.getListSortMap());
         options.putReferenceConfig(compareConfig.getReferenceMap());
         options.putDecompressConfig(compareConfig.getDecompressConfig());
+
+        if (Objects.equals(category, MockCategoryType.DATABASE.getName())) {
+            options.putExclusions(ignoreInDataBaseMocker);
+        }
         return options;
     }
 
+    private int convertCategoryType(String category) {
+        if (Objects.equals(category, MockCategoryType.DATABASE.getName())) {
+            return CategoryType.DATABASE;
+        }
+        return CategoryType.NORMAL;
+    }
```

Comment:
Please remove the category convert mapping and use String category directly.

Reply:
done